frameworks/elastic/universe/config.json

```diff
@@ -34,6 +34,11 @@
           "description": "Elasticsearch deploy strategy. [parallel, serial]",
           "type": "string",
           "default": "parallel"
+        },
+        "mesos_api_version": {
```

Comment:
Looks like there's supposed to be a `"MESOS_API_VERSION": "{{service.mesos_api_version}}"` in marathon.json. See examples in kafka, helloworld, cassandra.